misk-hotwire/src/main/kotlin/misk/hotwire/BuildHtml.kt

```diff
@@ -0,0 +1,15 @@
+package misk.hotwire
+
+import kotlinx.html.TagConsumer
+import kotlinx.html.stream.appendHTML
+
+fun buildHtml(renderer: TagConsumer<*>.() -> Unit) = StringBuilder().apply {
+  appendHTML().renderer()
+}.toString()
+
+// TODO replace with ResponseBody
```

Comment:
TODO?

Reply:
Addressed!

Reply:
I'm going to pair with Jesse on finalizing this since it was a point he had raised in initial prototypes.